cea/interfaces/dashboard/inputs/static/css/deckgl.css

```diff
@@ -0,0 +1,16 @@
+#tooltip:empty {
+  display: none;
+}
+#tooltip {
```

Comment:
this is probably where you can specify that the tooltip is to be moved a bit to the side. idk. also, might not be the actual issue, just my guess.